Sidney-/slugtrade
# Testing

New functionality:

- Added framework for automated testing.
- Modified requirements.txt to include selenium for testing.
- Added web drivers for Chrome and Firefox to repo.

How to test:

- Ensure that you have Chrome installed on your machine.
- Run venv_setup.sh to update your local repo's python environment.
- Initialize the server by running renew_db.sh.
- Start the server by running start.sh.
- Run the script run_tests.sh; Chrome will start automatically and drive through the tests.
- Check the output of run_tests.sh: one test called "test_title" ran and its status was "PASS".
